src/Form/Checkbox/index.js

```diff
@@ -18,11 +17,10 @@ const Checkbox = React.forwardRef(({
    label,
    ...props
 }, ref) => {
-   const oioContext = useContext(OIOContext)
    const formContext = useContext(OIOFormContext)
 
    const inputAppearance = appearance || formContext.elementAppearance
-   const inputHighlightColor = highlightColor || oioContext.highlightColor
+   const inputHighlightColor = highlightColor || formContext.highlightColor
```

Comment:
Just curious, how come you removed the `oioContext.highlightColor`, what if the `Checkbox` is not used in a form? Why not do:

```js
const inputHighlightColor = highlightColor || formContext.highlightColor || oioContext.highlightColor
```